java/connecting-to-bigtable/src/main/java/com/example/cloud/bigtable/connecting/BigtableHelperWithConfiguration.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START bigtable_connecting_helper_with_config]
+package com.example.cloud.bigtable.connecting;
+
+import java.io.IOException;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hbase.HBaseConfiguration;
+import org.apache.hadoop.hbase.client.Connection;
+import org.apache.hadoop.hbase.client.ConnectionFactory;
+
+public class BigtableHelperWithConfiguration {
+
+  public static Connection connection = null;
+
+  public static void connect() throws IOException {
+    // [START bigtable_creating_connection_object]
+    Configuration config = HBaseConfiguration.create();
```

Comment:
A comment would be useful here to why you would use a config or an example of a config argument.

Reply:
Ok, so this picks up config from `hbase_site.xml` ? (In that case that would be useful to know as a comment.

Reply:
Done